conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptions.java

```diff
@@ -79,10 +80,8 @@ public void handle(HttpServerExchange exchange, Throwable throwable) {
             deadlineExpiredException(exchange, (DeadlineExpiredException) throwable);
         } else if (throwable instanceof Error) {
             error(exchange, (Error) throwable);
-        } else if (throwable instanceof IOException && !exchange.getConnection().isOpen()) {
-            log.info(
-                    "I/O exception from a closed connection. The request may have been aborted by the client",
-                    throwable);
+        } else if (throwable instanceof IOException) {
+            ioException(exchange, (IOException) throwable);
```

Comment:
nit: can use instanceof pattern variable to avoid cast:

```suggestion
        } else if (throwable instanceof IOException ioException) {
            ioException(exchange, ioException);
```

Reply:
Let's adopt this convention for all the branches here for consistency.